mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -6054,6 +6054,31 @@ func TestAccContainerCluster_autopilot_withAdditiveVPC(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_cloudDns_nil_scope(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withAdvancedDNSConfig(clusterName, false, true, true, false, ""),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_container_cluster.primary", "dns_config.0.cluster_dns_scope", ""),
+				),
```

Comment:
We generally don't do this kind of check - it (more or less) duplicates work done by the following ImportState step.

```suggestion
```